server/src/main/java/org/opensearch/indices/replication/PrimaryShardReplicationSource.java

```diff
@@ -78,14 +80,28 @@ public void getSegmentFiles(
     ) {
         final Writeable.Reader<GetSegmentFilesResponse> reader = GetSegmentFilesResponse::new;
         final ActionListener<GetSegmentFilesResponse> responseListener = ActionListener.map(listener, r -> r);
+        // Few of the below assumptions and calculations are added for experimental release of segment replication feature in 2.3
+        // version. These will be changed in next release.
```

Comment:
nit: `These will be changed in next release.` ->  `These can change in upcoming releases.` ?

Reply:
Sure I can update that quickly